Scripts/script-CommonServerPython.yml

```diff
@@ -182,13 +182,13 @@ script: |-
       first = None
       if date1 == 'now':
           first = datetime.now(tzlocal())
-      elif isinstance(date1, basestring):
+      elif is_str_or_unicode(date1):
```

Comment:
why not just using:
`isinstance(s, STRING_TYPES)`

Reply:
Because I didn't know of it. Awesome feedback, will switch implementation